gke-deploy/core/image/image.go

```diff
@@ -47,31 +40,32 @@ func ParseImages(ctx context.Context, images []string) ([]Image, error) {
 			return nil, fmt.Errorf("duplicate image name: %q", imName)
 		}
 		exists[imName] = true
-		ims = append(ims, im)
+		refs = append(refs, ref)
 	}
 
-	return ims, nil
+	return refs, nil
 }
 
-func parseImage(ctx context.Context, image string) (Image, error) {
+func parseImage(image string) (name.Reference, error) {
```

Comment:
This method doesn't really add much. It adds a small prefix to the error string, but I'd argue that's not enough to justify having this method. Just call `name.ParseReference` directly in `ParseImages` and remove this method.